core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2TokenExchangeResource.java

```diff
@@ -114,13 +114,16 @@ private static Response pendingResponse(HttpServletRequest request)
     @ResourceSecurity(PUBLIC)
     @DELETE
     @Path("{authId}")
-    public void deleteAuthenticationToken(@PathParam("authId") UUID authId)
+    public Response deleteAuthenticationToken(@PathParam("authId") UUID authId)
     {
         if (authId == null) {
             throw new BadRequestException();
         }
 
         tokenExchange.dropToken(authId);
+        return Response
+                .ok()
```

Comment:
Maybe return some `ok("Authentication dropped", Text)` as for no content 204 should be returned as it is right now.

Reply:
I don't think that's necessary. Human does not consume this endpoint and we check only for http status